cico_functions.sh

```diff
@@ -33,6 +33,27 @@ function load_jenkins_vars() {
   fi
 }
 
+function check_buildx_support() {
+  export DOCKER_BUILD_KIT=1
```

Comment:
I think set up this env variables should be in the install_deps() method. As for me it's looks like installation part.

Reply:
yes sure will change and update the PR.

Reply:
yes sure will change and update the PR.